src/gui/advancedsettings.cpp

```diff
@@ -815,6 +822,11 @@ void AdvancedSettings::loadAdvancedSettings()
     // Tracker port forwarding
     m_checkBoxTrackerPortForwarding.setChecked(pref->isTrackerPortForwardingEnabled());
     addRow(TRACKER_PORT_FORWARDING, tr("Enable port forwarding for embedded tracker"), &m_checkBoxTrackerPortForwarding);
+#if defined(Q_OS_MACOS) || defined(Q_OS_WIN)
+    // Mark-of-the-Web
+    m_checkBoxMarkOfTheWeb.setChecked(pref->isMarkOfTheWebEnabled());
+    addRow(ENABLE_MARK_OF_THE_WEB, tr("Enable Mark-of-the-Web (MOTW) for downloaded files"), &m_checkBoxMarkOfTheWeb);
```

Comment:
Wouldn't it be better to use platform specific explanation here? Wouldn't macOS users be confused otherwise?

Reply:
OK done.